src/routes/online-lecture.js

```diff
@@ -41,6 +34,7 @@ export default () => {
     }
   };
 
+  let randomStr = (Math.floor(Math.random() * 50) + '').repeat(Math.floor(Math.random() * 30));
```

Comment:
```suggestion
  let randomStr = Math.random().toString(36).slice(2);
```

랜덤 함수는 간단하게 많이 사용되는 [36진법 트릭](https://velog.io/@awesomelon/%EA%B0%84%EB%8B%A8%ED%95%98%EA%B2%8C-%EB%82%9C%EC%88%98%ED%99%94-%EB%AC%B8%EC%9E%90%EC%97%B4-%EC%83%9D%EC%84%B1%ED%95%98%EA%B8%B0)을 이용해서 만들어주면 더욱 좋을 것 같습니다.

![image](https://user-images.githubusercontent.com/50603255/235338578-156327d9-8c33-4411-8a92-457e56f74a19.png)